backend/REST-API.js

```diff
@@ -5,20 +5,13 @@ const {
   allPoems
 } = require('./rest-api-test-data.js')
 const bodyParser = require('body-parser')
-// const addPoem = require('./rest-api-test-new-data.js')
 const { addPoem } = require('./init_database.js')
```

Comment:
we should import the `allPoems` function from this file `init_database.js` instead
```js
const { addPoem, allPoems } = require('./init_database.js')
```
... problem is there is no `allPoems` function in that file, so we'll need to create it (see below)